libcontainer/configs/config.go

```diff
@@ -21,10 +21,15 @@ type IDMap struct {
 
 // Seccomp represents syscall restrictions
 type Seccomp struct {
-	DefaultAction Action     `json:"default_action"`
-	Syscalls      []*Syscall `json:"syscalls"`
+	DefaultAction      Action     `json:"default_action"`
+	AddedArchitectures []Arch     `json:"added_architectures"`
+	Syscalls           []*Syscall `json:"syscalls"`
 }
 
+// An architecture whose syscalls will be allowed by Seccomp
+// By default, only the native architecture of the kernel is allowed
+type Arch string
```

Comment:
Why not just string instead of making a type?  

You usually only make a type like this if you are going to add methods to it.  I don't think we need a type here.